Formula/a/apify-cli.rb

```diff
@@ -0,0 +1,27 @@
+require "language/node"
+
+class ApifyCli < Formula
+  desc "Apify command-line interface"
+  homepage "https://docs.apify.com/cli"
+  url "https://registry.npmjs.org/apify-cli/-/apify-cli-0.18.0.tgz"
+  sha256 "28d4e8b032992560a4f842a44e8d107693b1a031560325b90967245c4ed7b628"
+  license "Apache-2.0"
+
+  depends_on "node@20"
```

Comment:
Do I have to use `node` instead of `node@20`? I'd rather pin the major Node.js version instead of always using the latest one, with which the CLI might not work.

Reply:
If you write a good test maintainers won't automatically move the app to the new node version if it is broken. Homebrew rules say that software should use the latest working version, so your preference doesn't really factor into it.

Reply:
That makes sense, but in `homebrew/core`, we try to use the newest dependencies when possible. In case of build/test failure, we can migrate to an older version of `node` as needed. (We'll discover that, because dependents are tested when a formula is updated to a newer version -- and that includes `node`. And hence the importance of tests.)

Reply:
Alright, thanks.